scripts/utils/package/package.sh

```diff
@@ -0,0 +1,50 @@
+# shellcheck shell=sh
+# shellcheck disable=SC2039,SC2059
+
+# Title:          package
+# Description:    A library for package utils.
+# License-SPDX:   MIT
```

Comment:
Remove the `MIT` line. Should release it as Apache as rest of the build infrastructure.